src/Blocks/BlockEnderchest.h

```diff
@@ -8,9 +8,9 @@
 
 
 class cBlockEnderchestHandler :
-	public cYawRotator<cBlockEntityHandler>
+	public cYawRotator<cContainerEntityHandler<cBlockEntityHandler>, 0x07, 0x03, 0x04, 0x02, 0x05>
```

Comment:
As weird as this may sound, Enderchests aren't "Container Entities" because the ender chest inventory is stored in the player and not the chest. The `cYawRotator` fix on its own should be enough.